Example/Tests/Classes/LeanplumTest.m

```diff
@@ -1781,4 +1782,22 @@ - (void)on_start_response:(BOOL) success
     XCTAssertTrue(success);
 }
 
+/**
+ * Test trigger message displayed calls callback
+ */
+-(void)test_triggerMessageDisplayed
```

Comment:
This tests callback, not "displayed". Misnomer? Also can we have a test that the callback gives all of the business requirements for the message?